Gemfile

```diff
@@ -130,6 +130,8 @@ group :development, :test do
   gem 'bullet', '>= 4.14.9'
 
   gem 'parallel_tests'
+  gem 'launchy'
+  gem 'pry-byebug'
```

Comment:
I am OK to put debugging tools in since most of us use it.

but maybe just `byebug` ? @weiqingtoh @fonglh  Any comments ?